osu.Game/Overlays/SettingsToolboxGroup.cs

```diff
@@ -156,6 +156,10 @@ protected override bool OnInvalidate(Invalidation invalidation, InvalidationSour
 
         private void updateExpandedState(ValueChangedEvent<bool> expanded)
         {
+            // clearing transforms is necessary to avoid a previous height transform
+            // potentially continuing to get processed while content has changed to autosize.
+            content.ClearTransforms();
```

Comment:
This makes me feel a bit uneasy. I'd clear specifically just the height transforms (`ClearTransforms(false, nameof(Height))`) just to be safe.

Reply:
On a second look, I suppose this is fine for the time being since it doesn't propagate by default.